src/js/export.js

```diff
@@ -0,0 +1,238 @@
+(function (){
+  "use strict";
+
+  window.export = {
+    toFile:exportCollectionToFile
+  }
+
+  var cID;
+  var cName;
+  var filename;
+  var CSVDelimiter;
+  var leafDelimiter;
+
+  var delimiters = [
+      {value: ',' , display: 'Comma (,)'},
+      {value: ';' , display: 'Semi-Colon (;)'},
+      {value: ':' , display: 'Colon (:)'},
+      {value: '|' , display: 'Pipe (|)'},
+      {value: '^' , display: 'Caret (^)'},
+      {value: '~' , display: 'Tilde (~)'},
+      {value: '\t' , display: 'Tab'},
+      {value: ' ' , display: 'Space'}
+  ];
+
+  function exportCollectionToFile(collID, colName){
+    cID = collID;
+    cName = colName;
+    var modal = createExportModal();
+
+    // Comma should probably not be the default delimiter
+    // since a lot of the entries contain commas.
+    selectDelimiterCSV.selectedIndex = 1;
+    selectDelimiterLeaf.selectedIndex = 3;
+
+    closeBtn.addEventListener('click', function() {
```

Comment:
This is not encouraged (id=closeBtn -> window/document.closeBtn), though it is a bit more safe because of "strict mode". Instead try to refactor such that you either supply onClick callbacks or set up the call chain yourself.